packages/components/docs/QueryModel.md

```diff
@@ -0,0 +1,165 @@
+results, add filters, add sorts, etc.) as well as add new models. The `queryModels` prop is an object that contains all
+the `QueryModel` objects you have defined, the keys of the object are the model ids, the values of the object are the
+`QueryModel` instances.
+
+The wrapped components (returned from `withQueryModels`) will take the same props you defined, except `actions` and
+`queryModels`. The `withQueryModels` HOC replaces `actions` and `queryModels` with several other props as defined by the
+interface `MakeQueryModels`:
+- `queryConfigs`: an object containing the queryConfigs for the models you want `withQueryModels` to instantiate. This
+prop is optional.
+- `autoLoad`: If autoLoad is true withQueryModels will automatically load the models when the component mounts. Defaults
+to false.
+- `modelLoader`: You can override how `QueryModels` are loaded by providing your own `QueryModelLoader`, you will not
+typically need to use this. It is primarily useful for testing.
+
+Below is set of examples showing off how withQueryModels is typically used.
+
+```tsx
+// First define the component you're going to wrap with withQueryModels
+interface MyComponentProps {
+    title: string;
+}
+
+// Here we create a type that includes InjectedQueryModels because we're wrapping the component with withQueryModels
+// which will inject queryModels and actions objects.
+type Props = MyComponentProps & InjectedQueryModels;
+
+// Here we use the name ExampleComponentImpl, users will not use this component directly, only the wrapped version below
+// which we expose to users as ExampleComponent.
+class ExampleComponentImpl extends PureComponent<Props> {
+    render() {
+        const { actions, queryModels, title } = this.props;
+        const model = queryModels.model;
+
+        return (
+            <div className="example-component">
+                <div>{title}</div>
+                <div>
+                    {model.isLoading && <LoadingSpinner />}
+                    {!model.isLoading && <span>The model has {model.rowCount} rows</span>}
+                </div>
+            </div>
+        );
+    }
+}
+
+// Next wrap your component with withQueryModels, here we set the type to MyComponentProps so the returned component,
+// ExampleComponent, can be used in a type safe manner. In this case, if the user forgets to pass in a title we'll get
+// a compiler error as intended.
+export const ExampleComponent = withQueryModels<MyComponentProps>(ExampleComponentImpl);
+
+// The component returned from withQueryModels, ExampleComponent in this case, now has the following props type:
+// MyComponentProps & MakeQueryModels. To use the component you pass it a queryConfig and it will isntantiate the
+// models you want and pass them to ExampleComponentImpl:
+const queryConfigs = { model: { schemaQuery: SchemaQuery.create('assay.general.amino acids', 'Runs') }};
```

Comment:
Suggest to use a more specific model name such as "mixture" instead of "model". I thought 'model' is a prop name for some interface when I first looked at the example, especially since it's queried using queryModels.model in ExampleComponentImpl